types/v56/records.go

```diff
@@ -0,0 +1,355 @@
+package types
+
```

Comment:
Need a comment about the purpose of this file and how it compares to the types.go file.

Reply:
removed this change, as Giuseppe made a lot changes there in his PR